include/javabind/binding.hpp

```diff
@@ -374,4 +375,46 @@ namespace javabind
             return *this;
         }
     };
+
+    struct EnumBinding {
+        std::vector<std::string_view> names;
+
+        void add(std::string_view name)
+        {
+            if (!contains(name)) {
+                names.push_back(name);
+            }
```

Comment:
I would suppose declaring the same enumeration value type twice would constitute an error.

Reply:
I currently allow this because the implementation supports mapping multiple native enums to a single java enum. Reporting an error here would mean this isn't supported anymore.

Mapping multiple enums to a single Java enum seems like an unlikely use case though. What's your preference, reporting an error here or allowing it?